### DISCUSSION TITLE
Quick Preview of FOLDER

### DISCUSSION THREAD

**Discussion by Erimus-Koo** [doublecmd/doublecmd#1605] [Suggestions]
While I use Totalcmd and move my cursor on a folder, then press ctrl+Q, the preview will list all the files in the folder, but in Double Commander, there's only the folder's full path.

The screenshot from Totalcmd.
![image](https://github.com/doublecmd/doublecmd/assets/17139670/3831dfa5-804b-4dd2-acd3-ed6d033f99aa)

The screenshot from Doublecmd.
<img width="293" alt="image" src="https://github.com/doublecmd/doublecmd/assets/17139670/8bafb1f6-1e50-4ae8-9329-1c0c2e98210c">

---

Double Commander
Version: 1.1.13 gamma
Revision: 453
Commit: f60bf09df
Build date: 2024/04/20
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 14.4.1

**Comment by alexx2000** [comment]
It is not standard TC function. It is `htmlview` plugin feature.

**Reply by Erimus-Koo** [reply]
Is it possible to install this plugin in DC on Mac?

**Reply by alexx2000** [reply]
No, it is a Windows only plugin.

**Reply by Erimus-Koo** [reply]
Got it, I'm new here, and may I ask one more question?
I saw many plugins in the [Plugins · doublecmd/doublecmd Wiki](https://github.com/doublecmd/doublecmd/wiki/Plugins), but how do I know which can be used on a M-chip Mac?

I've tried to download and compile but failed. After installing the lazbuild, but still something lost.

And from the "about" page:
```
Double Commander
Version: 1.1.13 gamma
Revision: 453
Commit: f60bf09df
Build date: 2024/04/20
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 14.4.1
```
The Lazarus and FPC look like already on my Mac, but I can't find them.

**Reply by alexx2000** [reply]
Some of them are cross-platform so they can be compiled under macOS. What plugin do you want to compile?
